### ISSUE TITLE
Alt text for journal thumbnail is not used on site index page

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#4048]
On the site-wide index page, journals are displayed with their journal thumbnails. Each thumbnail should have alt text, but the current default templates do not use this properly. Currently, the code looks like:

```
							{assign var="altText" value=$journal->getLocalizedData('journalThumbnailAltText')}
							<div class="thumb">
								<a href="{$url|escape}">
									<img src="{$journalFilesPath}{$journal->getId()}/{$thumb.uploadName|escape:"url"}"{if $altText} alt="{$altText|escape}"{/if}>
								</a>
							</div>
```

This should actually be:

```
							<div class="thumb">
								<a href="{$url|escape}">
									<img src="{$journalFilesPath}{$journal->getId()}/{$thumb.uploadName|escape:"url"}"{if $thumb.altText} alt="{$thumb.altText|escape}"{/if}>
								</a>
							</div>
```

I believe this is also an issue in the Bootstrap theme and may be in other themes.

**Comment by NateWr** [comment]
PR:
https://github.com/pkp/ojs/pull/2117

**Comment by NateWr** [comment]
@Vitaliy-1 did you have a chance to check if this same issue exists in the Bootstrap theme or any of the other official themes?
